api/presenter/service_offering.go

```diff
@@ -54,3 +59,10 @@ func ForServiceOffering(serviceOffering repositories.ServiceOfferingRecord, base
 		},
 	}
 }
+
+func ForServiceOfferingWithIncluded(serviceOffering repositories.ServiceOfferingRecord, baseURL url.URL, includes ...model.IncludedResource) ServiceOfferingWithIncludedResponse {
```

Comment:
I would suggest adding the `Included` field to the `ServiceOfferingResponse` type rather than coming up with a new one.

Reply:
At first I took this approach, but this would mean that we would have to change the way it is handled in the list endpoint as well - https://github.com/cloudfoundry/korifi/blob/main/api/handlers/service_offering.go#L72. There we handle the "includes" with the ForList presenter and including it in the ForServiceOffering() would change ForList as well. Since only the LIST and GET endpoints for service offerings include fields, it did not seem right - https://v3-apidocs.cloudfoundry.org/version/3.180.0/index.html#fields-parameter. What do you think ?

Reply:
Well, if the field is annotated with `omitempty`, then I believe it should be not visible in the response. If I am correct, then 
* the `get` handler **would** set it on the `ServiceOfferingResponse` object
* the `list` handler **would not**, it would use the includes as `ForList` argument. As the `Includes` would not be set on the `ServiceOfferingResponse` object, they would be just omited in the response resources. TL;DR I believe that changing the list endpoint would not be necessary.